pvlib/iotools/pvgis.py

```diff
@@ -665,3 +666,43 @@ def read_pvgis_tmy(filename, pvgis_format=None, map_variables=None):
         data = data.rename(columns=VARIABLE_MAP)
 
     return data, months_selected, inputs, meta
+
+
+def get_pvgis_horizon(latitude, longitude, url=URL, **kwargs):
+    r'''
+    Get horizon data from PVGIS
+
+    Parameters
+    ----------
+    latitude : float
+        Latitude in degrees north
+    longitude : float
+        Longitude in degrees east
+    url: string
+        Base URL for PVGIS
+    kwargs:
+        Passed to requests.get
+
+    Returns
+    -------
+    df : pd.DataFrame
+        Pandas dataframe of the retrived horizon
```

Comment:
The power of the iotools is in standardization, so I am all for changing it to the standard format, i.e., degrees from north clockwise.

My main concern was that it would be nice to be able to easily pass in the horizon data to the PVGIS functions, e.g., ``get_pvgis_hourly``. However, it doesn't seem like changing the azimuth convention would be an obstacle at all. In fact, it seems preferable to have the azimuth in degrees from north for this application. Here's the description of how to pass in a user horizon to PVGIS using the ``userhorizon`` parameter:

> Height of the horizon at equidistant directions around the point of interest, in degrees. Starting at north and moving clockwise. The series '0,10,20,30,40,15,25,5' would mean the horizon height is 0° due north, 10° for north-east, 20° for east, 30° for south-east, etc.

Reply:
Nice, that's convenient.  There is still the inconsistency with `aspect`, which uses 0=south, 90=west, -90=east.  Maybe we should change that too (not in this PR).

Reply:
@kandersolar I suppose you refer to the ``surface_azimuth`` parameter in ``get_pvgis_hourly`` - but yes I agree that it should be changed (must be a silly person who made that to begin with 😄 )

If we switch to north=0 we need to make sure the azimuth remains within 0-360 degrees of course (right now it's -180 to 180).

Reply:
Ok, I shifted the `horizon_azimuth` to use pvlib's convention. We could also put in a flag or some type of user option as well